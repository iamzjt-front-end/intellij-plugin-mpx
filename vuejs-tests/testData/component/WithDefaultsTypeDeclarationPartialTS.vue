<script setup lang="ts">
<error descr="Invalid number of arguments, expected 2">withDefaults</error>(defineProps<{
  stillProp: number,
}>());

stillProp; // TODO Should be unresolved
</script>

<template>
  {{stillProp}}
</template>
