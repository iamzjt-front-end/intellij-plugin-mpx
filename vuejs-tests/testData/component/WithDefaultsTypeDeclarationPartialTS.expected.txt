{
 class: "VueSourceComponent",
 components: {
  WithDefaultsTypeDeclarationPartialTS: <recursive value of class VueSourceComponent>,
 },
 props: [
  {
   class: "VueScriptSetupInputProperty",
   name: "stillProp",
   jsType: "number",
   required: true,
  },
 ],
}
